src/components/DebuggingTool/FlowChartNodes.tsx

```diff
@@ -34,26 +61,26 @@ export const InputNode = ({ data }: { id: string; data: any }) => {
       <InputNodeContainer>
         <NodeTitle>Id: {data?.id}</NodeTitle>
         <NodeTextContainer>
-          {data.uri ? <NodeImagePreview src={data.uri} /> : <></>}
-          {data.mime_type ? <NodeText>{data.mime_type}</NodeText> : <></>}
-          {data.text ? <NodeText>{data?.text}</NodeText> : <></>}
+          {data.uri && <NodeMediaPreview uri={data.uri} />}
```

Comment:
All these nodes look very similar. I wonder if it's possible to refactor them into one component and then set the little css differences with a prop